src/pint/toa.py

```diff
@@ -1638,7 +1638,7 @@ def compute_posvels(self, ephem=None, planets=False):
             if c in self.table.colnames:
                 log.info("Column {0} already exists. Removing...".format(c))
                 self.table.remove_column(c)
-        for p in ("jupiter", "saturn", "venus", "uranus"):
+        for p in ("jupiter", "saturn", "venus", "uranus", "neptune"):
```

Comment:
Can this tuple go up top in a variable so it doesn't have to be repeated all these times?